<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="incButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>+1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>-1</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>incButton</sender>
   <signal>clicked()</signal>
   <receiver>lcdNumber</receiver>
   <slot>raise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>decButton</sender>
   <signal>clicked()</signal>
   <receiver>lcdNumber</receiver>
   <slot>lower()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
